src/OSPSuite.UI/Services/DataImporter.cs

```diff
@@ -169,5 +177,45 @@ DataImporterSettings dataImporterSettings
             return result;
          }
       }
+
+      public Cache<string, IEnumerable<DataRepository>> ReloadFromConfiguration(IEnumerable<DataRepository> dataSetsToImport,
```

Comment:
Can those input parameters be IReadOnly?

Reply:
done